NetMQ.ReactiveExtensions.Tests/project.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "version": "1.0.0-*",
+  "dependencies": {
+    "NetMQ.ReactiveExtensions": "1.0.0-*"
+  },
+
+  "testRunner": "nunit",
+
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.1"
+        },
+        "NUnit": "3.5.0",
+        "dotnet-test-nunit": "3.4.0-beta-3"
+      },
+      "imports": "dnxcore50"
+    },
+    "net45": {
+      "dependencies": {
+        "NUnit": "2.6.4",
+        "System.Xml.Linq": "*"
```

Comment:
I'm 99% sure this line is not needed. `net45` is the full framework, so there's no dependency on external packages to provide `System.Xml.Linq`.